createdisk.sh

```diff
@@ -380,7 +387,7 @@ create_qemu_image $libvirtDestDir
 
 copy_additional_files $1 $libvirtDestDir
 
-tar cJSf $libvirtDestDir.$crcBundleSuffix $libvirtDestDir
+create_tarball $libvirtDestDir
```

Comment:
This can be `tar cSf - $libvirtDestDir | xz --threads=0 >$libvirtDestDir.$crcBundleSuffix`

Reply:
@cfergeau is there any improvement using this over executing the commands one by one, just curious :)

Reply:
Main improvement is that you don't need an intermediate file that you need to remove afterwards. But not a big difference between the 2.